update readme (#12)
lambdaclass/fuzzing_examples: README.md

 * [honggfuzz](https://github.com/google/honggfuzz)
 * [cargofuzz](https://github.com/rust-fuzz/cargo-fuzz)

Fuzzers structure 
---

In this repo the fuzzers have the following structure:

> ### if the fuzzer uses honggfuzz 

Reviewer: ```suggestion
### Honggfuzz' fuzzer
```

Author: done!

* The  ***src*** folder contains the file main with the fuzzer 

> * The ***hfuzz_workspace*** folder contains a REPORT file with the report of the crashes the fuzzer found, this folder is updated each time the fuzzer finds a new crash and has the information about the crash along with the name of the input that triggered it. Also contains the input files that triggers the errors, the file names starts with SIGABRT

Reviewer: ```suggestion
* The ***hfuzz_workspace*** folder contains a REPORT file with the report of the crashes the fuzzer found, this folder is updated each time the fuzzer finds a new crash and has the information about the crash along with the name of the input that triggered it. Also contains the input files that trigger the errors, the file names start with SIGABRT
```

Author: done!

> ### if the fuzzer uses cargofuzz 

Reviewer: ```suggestion
### Cargofuzz' fuzzer
```

Author: done!

> * the ***fuzz_targets*** contains the files with the fuzzers
> 
> 

Reviewer: ```suggestion
* the ***fuzz_targets*** contains the files with the fuzzers

```

It has one too many new lines

Author: done!

## User Guide

### 1. create the docker container 

> * Use the command  `docker build -t ready-to-fuzz-image` inside this proyect to create a docker image named **ready-to-fuzz-image** from the dockerfile in the proyect.

Reviewer: ```suggestion

* Use the command  `docker build -t ready-to-fuzz-image` inside this project to create a docker image named **ready-to-fuzz-image** from the dockerfile in the project.
```

Author: done!

Reviewer: It should say 'project' not 'proyect'

> * run the command `docker run -it --rm --cap-add=SYS_PTRACE --security-opt seccomp=unconfined ready-to-fuzz-image bash` to run the container from the image previusly created.

Reviewer: ```suggestion
* run the command `docker run -it --rm --cap-add=SYS_PTRACE --security-opt seccomp=unconfined ready-to-fuzz-image bash` to run the container from the image previously created.
```

Author: done!

### 2. Run a fuzzer 

Depending on the fuzzer use the following commands:

### If the Fuzzer Uses Honggfuzz

> * to run it with total random data use: 

Reviewer: ```suggestion
* to run it with totally random data use: 
```

Author: ups typo, fixed!
